src/AzureDeprecation.Notices.Management/IssueFactory.cs

```diff
@@ -110,10 +116,13 @@ private static void WriteTimelineEntriesForAllPhases(List<InputTimeLineEntry> ti
         }
 
         private static void WriteRequiredAction(NewAzureDeprecationV1Message newNoticeV1MessageQueueMessage, StringBuilder issueBuilder)
-        {
-            issueBuilder.AppendLine("### Required Action");
-            issueBuilder.AppendLine();
-            issueBuilder.AppendLine(newNoticeV1MessageQueueMessage.RequiredAction.Description);
+        {
+            if (!string.IsNullOrWhiteSpace(newNoticeV1MessageQueueMessage.RequiredAction?.Description))

```

Comment:
Can you revert this please?

We should always create this section. If there is no description; we should add that in the 3rd append line

Reply:
Adjusted, but with TODO for proper wording